roles/static_files/templates/etc/nginx/sites-available/static_files

```diff
@@ -8,12 +8,15 @@ server {
     listen   8080;
     listen   [::]:8080 default ipv6only=on;  ## listen for ipv6
     port_in_redirect off;
+    expires          -1;
 
     location /specials {
+        expires    1m;
```

Comment:
specials change so rarely, we can make this much longer - Though expires only will control time till recheck. Setting _something_ is better than nothing.

Reply:
Per our discussion, after 1 minute the browser will simply recheck the etags and the request is answered by nginx, so it's fast enough that we should be able to keep this at 1 minute.